sonar-html-plugin/src/test/resources/checks/AriaProptypesCheck.html

```diff
@@ -0,0 +1,53 @@
+<!-- Non-aria attribute -->
+<div aria-foo="true" />
+<div foo="true" />
+
+<!-- Boolean -->
+<div aria-hidden="true" />
+<div aria-hidden="false" />
+<div aria-hidden="undefined" />
```

Comment:
should we support `undefined` in HTML? I mean it makes sense in JSX, but here?

Reply:
Indeed, it doesn't make any sense here.